docs/source/topics/proc_accessing-the-openshift-web-console.adoc

```diff
@@ -31,8 +31,8 @@ You can also view the password for the `developer` and `kubeadmin` users by runn
 $ {bin} console --credentials
 ----
 
-See link:{crc-gsg-url}#troubleshooting-codeready-containers_gsg[Troubleshooting {prod}] if you cannot access the {prod} OpenShift cluster.
+See link:{crc-gsg-url}#troubleshooting-codeready-containers_gsg[Troubleshooting {prod}] if you cannot access the {prod} cluster.
```

Comment:
ditto, just `{prod}`?

Reply:
Changing to "The {ocp} cluster managed by {prod}".